_data/authors.yml

```
@@ -255,6 +256,10 @@ authors:
     name: "Andrew Whitmell"
     author-summary: "Developer based in Newcastle."
     picture: awhitmell.jpg
+  bell-jones:
+    name: "Benjamin Ell-Jones"
+    author-summary:"Developer based in Bristol"
```

```suggestion
    author-summary: "Developer based in Bristol"
```
I assume the build is unhappy with the lack of a space here.